### ISSUE TITLE
xmore notifications not behaving correctly

### ISSUE THREAD

**Issue by fwSmit** [dunst-project/dunst#915]
![image](https://user-images.githubusercontent.com/5826834/133913831-08e8ce58-43b0-4d68-aec0-98a0b5473eba.png)
There's at least two issues:

- The xmore notification shows an icon an a progress bar
- The hidden notifications don't time out.

**Comment by WhitePeter** [comment]
Could #910 have anything to do with it? Just asking because it changed the default lookup for system-wide dunstrc.

>     * The hidden notifications don't time out.

Should they, though? I would not want to miss notifications, just because the timeout ran while in hidden state.

**Comment by fwSmit** [comment]
> Could #910 have anything to do with it? Just asking because it changed the default lookup for system-wide dunstrc.

I don't think so. I think this behavior was already there when I added the progress bar.


> >     * The hidden notifications don't time out.
> 
> Should they, though? I would not want to miss notifications, just because the timeout ran while in hidden state.

You're right, they shouldn't time out. I'll edit the issue description.
